tests/unit/accordion/accordion_core.js

```diff
@@ -7,15 +7,23 @@ module( "accordion: core", setupTeardown() );
 
 $.each( { div: "#list1", ul: "#navigation", dl: "#accordion-dl" }, function( type, selector ) {
 	test( "markup structure: " + type, function() {
```

Comment:
I don't like that the structure test was changed to check functionality of the classes option. I'd prefer classes were handled in a different test and fully tested with changes to the option.